src/commands/health-check.ts

```diff
@@ -0,0 +1,37 @@
+import {flags} from '@oclif/command'
+import * as colors from 'colors'
+import Constants from '../services/constants'
+import healthCheck from '../utils/health-checker'
+import PercyCommand from './percy-command'
+
+export default class HealthCheck extends PercyCommand {
```

Comment:
I think you might just be able to extend `Command` instead due to the nature of this command.

Reply:
Really good idea -- there's a lot more in PercyCommand than I realized